core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -31,8 +32,8 @@ export class BubbleCluster extends Chart {
   constructor({
     selector,
     data,
-    width,
-    height,
+    width = 400,
```

Comment:
Maybe use a couple of constants? Like `width` and `ratio` (or `height` and  `ratio`)